File: app/src/main/java/org/wen/gank/di/AppComponent.java
package org.wen.gank.di;

import dagger.Component;
import javax.inject.Singleton;
import org.wen.gank.CategoryFragment;

/**
 * created by Jiahui.wen 2017-07-23
 */
@Singleton
@Component(modules = { AppModule.class })
public interface AppComponent {

    void inject(CategoryFragment target);
}

File: app/src/main/java/org/wen/gank/di/AppComponent.kt
package org.wen.gank.di

import dagger.Component
import org.wen.gank.CategoryFragment
import javax.inject.Singleton

/**
 * created by Jiahui.wen 2017-07-23
 */
@Singleton
@Component(modules = [AppModule::class])
interface AppComponent {

  fun inject(target: CategoryFragment)
}
